### DISCUSSION TITLE
How to use dashboard variables in filtering data from CSV

### DISCUSSION THREAD

**Discussion by jinishbshah** [grafana/grafana-csv-datasource#238] [Q&A]
I am trying to filter data using the dashboard variable dropdown but I need to know how can I use the variable selection in Transform to filter the data depending on that

**Comment by jholdorff** [comment]
I'm trying the same thing, too bad no one has the solution.
Several people worked so hard on this plugin and in the end users can't use it just because the functionality is not explained properly. Unfortunately this happens way too often.

**Comment by zoltanbedi** [comment]
This is not related to the plugin but to grafana. Please open an issue there.
